src/pymovements/plotting/main_sequence_plot.py

```diff
@@ -55,10 +61,19 @@ def main_sequence_plot(
     ----------
     events: Events | EventDataFrame | None
         It must contain columns "peak_velocity" and "amplitude".
+    fit: bool
```

Comment:
please move all fit-related arguments (`fit`, `fit_color`, `fit_measure`) after the `marker` argument such that they are all grouped together.